libs/s25main/network/GameServer.cpp

```diff
@@ -185,7 +186,7 @@ bool GameServer::Start(const CreateServerInfo& csi, const boost::filesystem::pat
     if(!mapinfo.mapData.CompressFromFile(mapinfo.filepath, &mapinfo.mapChecksum))
         return false;
 
-    bfs::path luaFilePath = bfs::path(mapinfo.filepath).replace_extension("lua");
+    bfs::path luaFilePath = lua_path.get_value_or(bfs::path(mapinfo.filepath).replace_extension("lua"));
     if(bfs::is_regular_file(luaFilePath))
```

Comment:
shouldn't it be an error if we explicitly want to load a lua file, but it does not exist? (e.g `lua_path` *not* empty but file does not exist)